solr/test-framework/src/java/org/apache/solr/SolrJettyTestBase.java

```diff
@@ -106,39 +116,16 @@ public static JettySolrRunner createAndStartJetty(String solrHome) throws Except
   public static JettySolrRunner createAndStartJetty(
       String solrHome, Properties nodeProperties, JettyConfig jettyConfig) throws Exception {
 
-    initCore(null, null, solrHome);
-
-    Path coresDir = createTempDir().resolve("cores");
-
-    Properties props = new Properties();
-    props.setProperty("name", DEFAULT_TEST_CORENAME);
-    props.setProperty("configSet", "collection1");
```

Comment:
this particular choice wasn't retained in your changes -- the use of "collection1" as the configSet when creating a collection.